schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "xref_list",
+            "datatype": "string"
+          },
+          {
+            "name": "citation_list",
+            "datatype": "string"
+          },
+          {
+            "name": "comment",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "attribute_set",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "variaiton_id",
+            "datatype": "integer"
+          },
+          {
+            "name": "number_of_copies",
+            "datatype": "integer"
+          }
+        ]
+      },
+      {
+        "name": "type_geno_type_scv",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "simple_allele",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "haplo_type",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "name",
+            "datatype": "string"
+          },
+          {
+            "name": "variation_type",
+            "datatype": "string"
+          },
+          {
+            "name": "functional_consequence",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "interpretations",
+            "datatype": "string"
+          },
+          {
+            "name": "xref_list",
+            "datatype": "string"
+          },
+          {
+            "name": "citation_list",
+            "datatype": "string"
+          },
+          {
+            "name": "comment",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "attribute_set",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "variaiton_id",
+            "datatype": "integer"
+          }
+        ]
+      },
+      {
+        "name": "submission_name_list",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "submission_name",
+            "datatype": "string",
+            "array_of": true
+          }
+        ]
+      },
+      {
+        "name": "trait_mapping_list",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "trait_mapping",
+            "datatype": "string",
+            "array_of": true
+          }
+        ]
+      },
+      {
+        "name": "trait_mapping",
+        "columns": [
+          {
+            "datatype": "string",
+            "datatype": "string"
+          },
+          {
+            "name": "med_ge
```

Comment:
Same question as `gene_list`

Reply:
Yes potential to hoist `type_scv` up as `array_of:true`?